react/features/chat/middleware.js

```diff
@@ -180,6 +177,28 @@ function _addChatMsgListener(conference, { dispatch, getState }) {
             });
         }
     );
+
+    conference.on(
+        JitsiConferenceEvents.CONFERENCE_ERROR, (error, ...params) => {
+            error === JitsiConferenceErrors.CHAT_ERROR && _handleChatError(store, params);
```

Comment:
Isn't params an array? If so, it's okay to pass an array to _handleChatError as the error?

Reply:
Updated, thanks